js/pricing.js

```diff
@@ -9,17 +12,40 @@ $(
     function () {
         const $confirmPersonalInformation = $('#confirm-personal-information');
         const $confirmDevelopmentAgreement = $('#confirm-development-agreement');
-        const $disabledButton = $('#order-now-btn');
+        const $orderButton = $('#order-now-btn');
+        const $redirectScreen = $('#redirectScreen');
+        const $loader = $('.redirect-loader', '#redirectScreen');
+        const $errorContainer = $('.redirect-error', '#redirectScreen');
+        const $tryAgain = $('#try-again');
 
-        $confirmPersonalInformation.change(function () {
-            confirmAgreement(this, $disabledButton);
+        const orderUrl = "{{site.data.payment_config.orderUrl}}";
+        const prodApiUrl = "{{site.data.payment_config.prodApiUrl}}";
+        const devApiUrl = "{{site.data.payment_config.devApiUrl}}";
+        // `window.mode` we setup in the `getting-help/service-section.html` file by Jekyll.
+        const apiUrl = "{{jekyll.environment}}" === "development" ? devApiUrl : prodApiUrl;
```

Comment:
Can we have a normal `if` clause and drop `prod` and `dev` variables? We don't really need them, right?